app/invocation/dense/dense_invocation_overview.tsx

```diff
@@ -46,11 +48,26 @@ export default class DenseInvocationOverviewComponent extends React.Component {
             />
           </div>
         </div>
-        <div className={`dense-invocation-status-bar ${this.props.model.getStatusClass()}`}>
-          <div>
-            {this.props.model.targets.length} {this.props.model.targets.length == 1 ? "target" : "targets"} on&nbsp;
-            {this.props.model.getStartDate()} at {this.props.model.getStartTime()} for{" "}
-            <span title={this.props.model.getDurationSeconds()}>{this.props.model.getTiming()}</span>
+        <div
+          className={`dense-invocation-status-bar ${this.props.model.getStatusClass()} ${
+            !isBazelInvocation && "dense-ci-run-status-bar"
+          }`}>
+          <div className="dense-invocation-status-bar-left">
+            {!isBazelInvocation && <div className="ci-run-status-icon">{this.props.model.getStatusIcon()}</div>}
+            <div>
+              {isBazelInvocation ? (
```

Comment:
I'm fine with showing / hiding things based on whether or not we're dealing with a CI run, but once we start making the UI look different - we might as well create separate components.

Can we either:
a) Include the success / failure in the status bar for all invocations 
b) Just hide the `dense-invocation-overview-grid-chunk`s below that we want to hide
c) Hide all of the `dense-invocation-overview-grid-chunk` that have zero values

Reply:
Went with option (a) - WDYT?

Reply:
Changed to option (b) per offline discussion